### ISSUE TITLE
More consistent top-left/bottom-right hook undershoot behavior for turned/reversed letters under italics.

### ISSUE THREAD

**Issue by jmcwilliams403** [be5invis/Iosevka#2199]
Below is a comparison of several letters with turned/reversed forms for comparison, as well as a double storey a.
```
ƹ𝼁cceeƨɛɛɐ
ʒɡↄɔəǝƨɜᴈa
```
Upright for comparison:
![image](https://github.com/be5invis/Iosevka/assets/37010132/dc9af4df-4a75-464e-b957-e4458420244b)
Italic:
![image](https://github.com/be5invis/Iosevka/assets/37010132/20b18149-b7ad-49ff-a6ac-92c84fb321d3)

For clarification, I don't have any problem with the appearance of the undershoot when present, but I just wish it were applied more consistently, whether that'd be universally undershooting or universally static (or any other solution you think would be better).

Basically I wish the characters in the first row had the same hook as each other, and also if the characters from the second row separately had the same hook as each other. Whether one row matches the other is not essential.

**Comment by jmcwilliams403** [comment]
Maybe the least disruptive solution would be to make the turning function either create an undershoot if it didn't exist, or remove it if it did.
For the mirroring function though, I actually like how it behaves already.

**Comment by be5invis** [comment]
The principle is, if a letter is considered as *turned*, then the top-left hook will be swashy under italics.
If a letter is considered as *reversed* (left-right mirroring) or is an individual letter, then the top-left hook will be normal (flat) under italics.
Taking examples of the `ↄɔ`. The first one is the lunate Sigma symbol and the second one is the turn c, so we have a swashy terminal for the second `ɔ` but not the first `ↄ`.

Resolve as by design.
